runtime/rasdump/trigger.c

```diff
@@ -555,7 +556,53 @@ prepareForDump(struct J9JavaVM *vm, struct J9RASdumpAgent *agent, struct J9RASdu
 		newState |= J9RAS_DUMP_TRACE_DISABLED;
 	}
 
-	if ((context->eventFlags & (J9RAS_DUMP_ON_GP_FAULT | J9RAS_DUMP_ON_ABORT_SIGNAL | J9RAS_DUMP_ON_TRACE_ASSERT)) == 0) {
+	/* Release vm access until this thread has the dumpKey and is ready to run. This will allow other threads to obtain exclusiveVMAccess in the meantime. */
+	if (vmThread && !vmThread->inNative) {
```

Comment:
I would prefer `NULL != vmThread` here, but the old-style is used throughout the file, so I won't insist.

Reply:
updated with that change